### DISCUSSION TITLE
学生认证一直通不过

### DISCUSSION THREAD

**Discussion by 66Xuzming** [community/community#51467] [GitHub Education]
You are unlikely to be verified until you have completed your with your full name as it appears in your academic affiliation document plus a short bio.一直显示这个

**Comment by WJC-POG** [comment]
1、尽可能使用学校邮箱
2、去学信网申请英文版的在校证明，可以通过拽托的方式进行上传
3、不要开代理，在学校进行申请，因为会需要定位

**Comment by samus-aran** [comment]
Hi @66Xuzming 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍
